src/RulesEngine/Models/Workflow.cs

```diff
@@ -1,25 +1,36 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System;
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 
 namespace RulesEngine.Models
 {
+    [Obsolete("Workflow is now Workflow - Workflow will be removed in next major version")]
+    [ExcludeFromCodeCoverage]
+    public class WorkflowRules : Workflow {
+    }
+
     /// <summary>
     /// Workflow rules class for deserialization  the json config file
     /// </summary>
     [ExcludeFromCodeCoverage]
-    public class WorkflowRules
+    public class Workflow
     {
+        /// <summary>
+        /// Gets the workflow Id.
+        /// </summary>
+        public int Id { get; set; }
+
         /// <summary>
         /// Gets the workflow name.
         /// </summary>
         public string WorkflowName { get; set; }
 
         /// <summary>Gets or sets the workflow rules to inject.</summary>
         /// <value>The workflow rules to inject.</value>
-        public IEnumerable<string> WorkflowRulesToInject { get; set; }
```

Comment:
instead of removing right away can we keep it like
``` c#
public IEnumerable<string> WorkflowRulesToInject { get { return WorkflowsToInject; } 
 set{ WorkflowsToInject = value } } 
```

with a deprecated attribute?